web-services/cached-results/src/main/java/datawave/webservice/results/cached/CachedResultsBean.java

```diff
@@ -203,7 +203,7 @@ public class CachedResultsBean {
     private CreatedQueryLogicCacheBean qlCache;
 
     @Inject
-    private QueryPredictor predictor;
+    private QueryPredictor<?> predictor;
```

Comment:
Unclear to me why, but this change appears to be breaking the webservice deployment, causing Weld/CDI to throw an "unsatisfied dependency" error when trying to inject an instance

Reply:
Yeah -- just removed it and got it to run.